rde/device.cpp

```diff
@@ -3,6 +3,13 @@
 #include <filesystem>
 #include <iostream>
 
+#ifdef OEM_AMD
```

Comment:
Commit message : This is really host power-off use case . It should be replaced to Hosst power off or not ready to accept Patch request window.